errors/invalid-relative-url-external-as.md

```diff
@@ -0,0 +1,43 @@
+# Invalid relative `href` and external `as` values
+
+#### Why This Error Occurred
+
+Somewhere you are utilizing the `next/link` component, `Router#push`, or `Router#replace` with a relative route in your `href` that has an external `as` value. The `as` value must be relative also or only `href` should be used with an external URL.
+
+Note: this error will only show when the `next/link` component is clicked not when only rendered.
+
+**Incompatible `href` and `as`**
+
+```jsx
+import Link from 'next/link'
+
+export default () => (
+  <>
+    <Link href="/invalid" as="mailto:john@example.com">
+      <a>Invalid link</a>
+    </Link>
+  </>
+)
+```
+
+**Compatible `href` and `as`**
+
+```jsx
+import Link from 'next/link'
+
+export default () => (
```

Comment:
This should be named

Reply:
Updated, also updated the original err.sh I used as a template